### ISSUE TITLE
Knives PDF Pop-Ups on PDPs

### ISSUE THREAD

**Issue by juanmriosv** [Century21Stores/GLOBAL#35]
[Right Knife for the Right Job.pdf](https://github.com/Century21Stores/GLOBAL/files/632090/Right.Knife.for.the.Right.Job.pdf)

Halaine is requesting a hyperlink in product copy that opens a lightbox with the PDF in it.

Copy to read:
"To find the Right Knife for the Right Job, click here" (Just the word "here" should be underlined & hyperlinked.)

List of Products:
7029-6399000 — https://www.c21stores.com/products/handknfesharpnr
7029-7030000 — https://www.c21stores.com/products/4stageknifeshrpnr
*7086-0425000 — https://www.c21stores.com/products/25inpeelingpro (testable as of 1/4)
7086-0483000 — https://www.c21stores.com/products/8pcssstkknfset
7086-1081000 — https://www.c21stores.com/products/4inspreader
7086-2086000 — https://www.c21stores.com/products/kitchenshrsblk
7086-3205000 — https://www.c21stores.com/products/8inprochef
7086-3260000 — https://www.c21stores.com/products/6utilitywvyedgepro
7086-3538000 — https://www.c21stores.com/products/7inprosantoku
7086-4456000 — https://www.c21stores.com/products/6inutility
7086-9169000 — https://www.c21stores.com/products/3pcslvrpntparing
7086-9537000 — https://www.c21stores.com/products/35inparingpro

https://www.c21stores.com/pages/jmprototype

**Comment by juanmriosv** [comment]
"Is there anyway we could put the attached PDF as a pop up on product pages? We want this to show on all Wusthof items kind of like a size guide for knives, but instead of where we normally have it, have it set up as a bullet in the copy description. Let us know if this is possible. " ( Halaine )

**Comment by AdenaG** [comment]
@juanmriosv Where is this visible? (I checked jm prototype).

**Comment by juanmriosv** [comment]
Hi @AdenaG I believe it was a mistake when I assigned it to you. Not ready for QA yet.

**Comment by juanmriosv** [comment]
Ready on JMPrototype @AdenaG.
Best,
Juan

**Comment by c21hornak** [comment]
@juanmriosv Halaine wanted just the "here" to be underlined and hyperlinked. Currently it seems like "click here" is hyperlinked but no underline.

7086-0425000 — https://www.c21stores.com/products/25inpeelingpro

**Comment by juanmriosv** [comment]
@c21hornak 
I added the underline.
If you are planning to use "click here" / "here"  I would recommend to use "click here" because: 

- A CTA usually has an "Action" (verb) value that suggest you to take an action.
- "here" is a bit tight to click with fingers

But, in my opinion, we should use something different than "click here" or "here" because:

- Puts effort on the "Mouse Mechanics" rather than the actual action about to be done.
	For example: "Click **here** to see the PDF"  vs "**VIEW PDF**"
- The text around "here" or "click here" actually explains what the users are clicking. CTA is not descriptive enough 
	For example: "Look at my photos, here"  vs "Look at my photos **at the beach**, **in NYC**, **in Boston**"
- It is also non descriptive enough for assistance technologies. Some of them use the text inside the anchor tag to describe the link. 

Do you want me to write something about this in more detail?


Juan

**Comment by c21hornak** [comment]
@juanmriosv I agree, however I want to deliver this today since you're out the rest of the week. Let's go with the "click here" being underlined for now and if you want to write something for Halaine next week that is ok.

**Comment by c21hornak** [comment]
@AdenaG ready to QA (cross-browser/cross-device).

**Comment by AdenaG** [comment]
PASS, except in IE 11. And as known, pdf is large, so load isnt immediate on mobile.

**Comment by AdenaG** [comment]
Pass, IE too. And re mobile, bc only one pdf provided, copy is barely legible on smaller mobile devices.

**Comment by c21hornak** [comment]
Approved to push to PROD @juanmriosv
